### PR TITLE
Some minor changes

### PR DESCRIPTION
**Pull request by thanhturin**
Add some minor change

### CODE REVIEW

File: Xcode13/Xcode13/Xcode13App.swift [hunk 1]
```diff
@@ -10,8 +10,10 @@ import SwiftUI
 @main
 struct Xcode13App: App {
     var body: some Scene {
+
```
Comment (thanhturin) [inline Xcode13/Xcode13/Xcode13App.swift:13]: Can keep it as before

File: Xcode13/Xcode13/ContentView.swift [hunk 2]
```diff
@@ -9,8 +9,9 @@ import SwiftUI
 
 struct ContentView: View {
     var body: some View {
-        Text("Hello, world!")
-            .padding()
+        Text("Hello world!")
```
Comment (thanhturin) [inline Xcode13/Xcode13/ContentView.swift:12]:
```suggestion
        Text("Hello world!!!!")
```

File: Xcode13/Xcode13/Xcode13App.swift (hunk 1 above)
Comment (thanhturin) [reply Xcode13/Xcode13/Xcode13App.swift:13]: Okay. I will do it later

File: Xcode13/Xcode13/ContentView.swift (hunk 2 above)
Comment (thanhturin) [reply Xcode13/Xcode13/ContentView.swift:12]: Good idea

**Review by thanhtashp — APPROVED** [review]

**Pull request closed without merging**